app/src/org/commcare/utils/PrivilegesUtility.java

```diff
@@ -0,0 +1,185 @@
+package org.commcare.utils;
+
+import org.commcare.modern.util.Pair;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+
+/**
+ * Helper methods for privilege data
+ *
+ * Created by ctsims on 8/7/2017.
+ */
+
+public class PrivilegesUtility {
```

Comment:
Can you add javadoc comments with @returns declarations to all of the methods in here? The fact that they're all just returning different configurations of `Pair`s and `String[]`s is really confusing without that